sql/mob_droplist.sql

```diff
@@ -31842,8 +31842,8 @@ INSERT INTO `mob_droplist` VALUES (4336,0,12448,10);
 INSERT INTO `mob_droplist` VALUES (4336,0,12704,60);
 INSERT INTO `mob_droplist` VALUES (4336,0,12832,10);
 INSERT INTO `mob_droplist` VALUES (4336,0,12960,10);
-INSERT INTO `mob_droplist` VALUES (4337,0,853,100);
-INSERT INTO `mob_droplist` VALUES (4337,0,1296,110);
+INSERT INTO `mob_droplist` VALUES (4337,0,853,280);
+INSERT INTO `mob_droplist` VALUES (4337,0,1296,1180);
```

Comment:
Drop rates in this table are expressed as fractions of 1000 with 1/1000 being `0.1%` you can't put drop rates over 100%.
For multiples of teh same item you have to use more than 1 line.